.github/workflows/json-schema-check.yaml

```diff
@@ -0,0 +1,19 @@
+name: json-schema-check
+
+on:
+  pull_request:
+    paths:
+      - "**/schema/*.schema.json"
+      - "**/config/*.param.yaml"
+
+jobs:
+  json-schema-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
```

Comment:
@ambroise-arm Do you need this part? (Confirming just in case.)

https://github.com/actions/checkout#usage says:

>     # The branch, tag or SHA to checkout. When checking out the repository that
>     # triggered a workflow, this defaults to the reference or SHA for that event.
>     # Otherwise, uses the default branch.
>     ref: ''

```suggestion
```

Reply:
I didn't try without, it may indeed work perfectly fine without.
I think I was just following https://github.com/autowarefoundation/autoware.universe/blob/00dfda5229a006041f084c7b9bbb40b96c2387b3/.github/workflows/pre-commit.yaml#L21

Reply:
> I think I was just following

I see. It's for modifying the branch because `pre-commit` has to push the changes.
If you just want to check out the pull request branch, you don't need it.